### ISSUE TITLE
make more gov proposals

### ISSUE THREAD

**Issue by okwme** [cosmos/gaia#978]
## Summary

i'd like to lower the barrier and stigma of making signaling proposals on the hub. Let's collect some ideas.

## Proposal

* Rename the gaia repo
* Submit our OKRs
* Ask if Cosmos wants the hub team to be more active on governance

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by faddat** [comment]
100000000%

<3

<3


<3


<3

YES

<3

PS:  It's governance-- I would like to suggest that the hub team define itself on the hub using governance, and then seek mandate to make executive decisions. 

I do not know if OKR's matter at this time-- instead, it may make sense to get mandate to set objectives, then measure key results.  

**THIS IS THE WAY**

One other thing-- don't ask.  

There's no point.  Instead just get highly engaged.  I really like this concept.  Keep moving until Gaia is running Gaia.  I don't know about changing her name, though.  Might vote no as I could no longer reference performance issues by describing Gaia as a fat lady.

**Comment by nooomski** [comment]
Closed by bot, opening again.

**Comment by faddat** [comment]
Clarification:

When I say that Gaia is running Gaia, I mean that @okwme should seek his mandate from Gaia in a governance proposal.   This way, Gaia can have her own blend of politics, instead of running externally.   I'd support, and I think that others would, too.

Gaia's leadership shouldn't be chosen by any of the mothership orgs-- icf, informal, tendermint--  gaia is perfectly capable of choosing her own leadership.

There's been unmistakable improvements in many aspects of the hub since Billy picked up the code base, and I find the repo easier to work with now. 

Basically, I'm saying that @okwme makes an excellent candidate as someone to lead Gaia for a while, approved by Gaia and given a budget by Gaia.

I think that OKR's and stuff, come after that.  Get the mandate from the chain herself, and let's see just how far we can take this.  As for the title of the role, I'd like to suggest "regent", meaning temporary king/queen. 

I guess it might not be perfect because I don't think regents are elected, but I do not think that Gaia should have a CEO.  

...but an accountable leader?

very yes.

**Comment by faddat** [comment]
PS:  this should not be seen as disapproval of the work that ICF / informal / tm has done on gaia

quite the opposite in fact

If gaia is ready to run herself from an admin perspective, this is more of a "mission accomplished" than anything else.

And I think that she's ready.

PS:  yes, gov does need some work and a UI.

**Comment by okwme** [comment]
have a series of proposals that will be rolling out soon.
1) CosmWasm on Hub
2) Reduce voting period

General attitude of encouraging more proposals has been well socialized I expect to see rate of proposals increase over the next 6 months. Going to close this issue because it's not being used to actually make this happen although the topic itself is not closed.
